src/open_llm_vtuber/routes.py

```diff
@@ -21,23 +25,73 @@ def init_client_ws_route(default_context_cache: ServiceContext) -> APIRouter:
     """
 
     router = APIRouter()
-    ws_handler = WebSocketHandler(default_context_cache)
+    broadcast_websockets: Set[WebSocket] = set()
+    ws_handler = WebSocketHandler(default_context_cache, broadcast_websockets)
+
+    async def process_queue(websocket: WebSocket):
+        try:
+            while True:
+                message = await websocket.receive_json()
+                # data = json.loads(message)
+                logger.debug(f"✨Received data from web: {message}")
+                await message_queue.put(message)
+
+        except Exception as e:
+            logger.error(f"Error : {e}")
 
     @router.websocket("/client-ws")
     async def websocket_endpoint(websocket: WebSocket):
         """WebSocket endpoint for client connections"""
         await websocket.accept()
         client_uid = str(uuid4())
 
+        logger.debug(f"client_uid: {client_uid}")
+
         try:
+            asyncio.create_task(process_queue(websocket))
+
             await ws_handler.handle_new_connection(websocket, client_uid)
-            await ws_handler.handle_websocket_communication(websocket, client_uid)
+            await ws_handler.handle_websocket_communication(
+                websocket, client_uid, message_queue
+            )
+
         except WebSocketDisconnect:
             await ws_handler.handle_disconnect(client_uid)
         except Exception as e:
             logger.error(f"Error in WebSocket connection: {e}")
             await ws_handler.handle_disconnect(client_uid)
-            raise
+
+    @router.websocket("/add_msg-ws")
```

Comment:
```suggestion
    @router.websocket("/add-msg-ws")
```
这个改成 `/add-msg-ws` 会不会好一些？(如果改的话其他地方可能也要改一下)

Reply:
要改的话,这里也改一下就行了

```python
async def broadcast_client():
    """
    监听队列并通过 WebSocket 发送消息 | Listen to the queue and send messages via WebSocket.
    """
    uri = "ws://localhost:12393/add_msg-ws"
```